documentation/dotnet-trace-instructions.md

```diff
@@ -35,6 +37,22 @@ Collecting to file: <Full-Path-To-Trace>/trace.netperf
 
 - Finally, stop collection by pressing the \<Enter> key, and *dotnet-trace* will finish logging events to *trace.netperf* file.
 
+## Viewing the trace captured from dotnet-trace
+
+On Windows, `.netperf` files can be viewed on PerfView (https://github.com/microsoft/perfview) for analysis, just like traces collected with ETW or LTTng. For traces collected on Linux, you can either move the trace to a Windows machine to be viewed on PerfView. 
```

Comment:
If my English grammar is correct, when you use the word either, you have to provide the or clause to provide another option.

Reply:
my English grammar is wrong.